build.gradle

```
@@ -12,7 +12,7 @@ buildscript {
         opensearch_group = "org.opensearch"
         opensearch_version = System.getProperty("opensearch.version", "1.3.10-SNAPSHOT")
         // 1.0.0 -> 1.0.0.0, and 1.0.0-SNAPSHOT -> 1.0.0.0-SNAPSHOT
-        opensearch_build = opensearch_version.replaceAll(/(\.\d)([^\d]*)$/, '$1.0$2')
+        opensearch_build = opensearch_version.tokenize('-')[0] + '.0-SNAPSHOT'
```

the comment above is no longer true if we always add `SNAPSHOT`, try this?
```suggestion
        opensearch_build = opensearch_version.replaceAll(/(\.\d+)([^\d]*)$/, '$1.0$2')
```

---

Thanks. that worked.